catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/handlers/CqlTransformHandler.java

```diff
@@ -436,6 +447,24 @@ private void attachFileToResponse(
     BinaryContent content =
         bundleContext.getService(queryResponseTransformer).transform(cqlQueryResponse, arguments);
 
+    if (content.getInputStream() == null) {
+      LOGGER.debug("Unable to transform content for mimeType: {}", content.getMimeTypeValue());
+      response.status(HttpStatus.BAD_REQUEST_400);
+      return ImmutableMap.of(
+          "message",
+          "Unable to transform content for mimeType: "
+              + content.getMimeTypeValue()
+              + ", may be missing required field(s)");
+    }
+
+    if (!CollectionUtils.isEmpty(
```

Comment:
✏️ You can use `CollectionUtils.isNotEmpty(...)` to make this read a bit easier. 

https://commons.apache.org/proper/commons-collections/apidocs/org/apache/commons/collections4/CollectionUtils.html#isNotEmpty-java.util.Collection-